charts/operator-wandb/charts/parquet/templates/deployment.yaml

```diff
@@ -32,6 +33,16 @@ spec:
         {{- include "parquet.podLabels" . | nindent 8 }}
         {{- include "parquet.labels" . | nindent 8 }}
       annotations:
+        {{- if .Values.fuse.enabled }}
+        {{- if eq $bucketProvider "gcs" }}
```

Comment:
is this going to drop a sidecar into the pod?

Reply:
probably not, just worth knowing if we see issues with pod creation in the future.